docs/next/components/mdx/includes/dagster/concepts/LoggingInDagsterUI.mdx

```diff
@@ -0,0 +1,51 @@
+When jobs are run, the logs stream back to the [**Run details** page](/concepts/webserver/ui#run-details) in real time. The UI contains two types of logs - **structured event** and **raw compute** - which you can learn about using the tabs below.
+
+<TabGroup>
+<TabItem name="Structured event logs">
+
+### Structured event logs
```

Comment:
A downside of tabs are that they make it more difficult for users to:
- Ctrl-F for content they're looking for,
- Mindlessly scroll and skim for content and examples that seem relevant.

I do both of these frequently when navigating docs.

Do we feel confident that the benefits outweigh that?

Reply:
I see your point about this page in particular - I was re-using a piece of content that had everything in tabs. I broke it apart so this page has flat sections with the copy instead of tabs.